lib/exception_notifier/mattermost_notifier.rb

```diff
@@ -0,0 +1,152 @@
+module ExceptionNotifier
+  class MattermostNotifier
+
+    attr_accessor :httparty
+
+    def initialize(options = {})
+      super()
+      @default_options = options
+      @httparty = HTTParty
+    end
+
+    def call(exception, options = {})
+      @options = options.merge(@default_options)
+      @exception = exception
+
+      @env = @options.delete(:env)
+
+      @application_name = @options.delete(:app_name) || Rails.application.class.parent_name.underscore
+      @gitlab_url = @options.delete(:git_url)
+      @username = @options.delete(:username) || "Exception Notifier"
+      @avatar = @options.delete(:avatar)
+
+      @channel = @options.delete(:channel)
+      @webhook_url = @options.delete(:webhook_url)
+      raise ArgumentError.new "You must provide 'webhook_url' parameter." unless @webhook_url
+
+      unless @env.nil?
+        @controller = @env['action_controller.instance'] || MissingController.new
+
+        request = ActionDispatch::Request.new(@env)
+
+        @request_items = { url: request.original_url,
+                           http_method: request.method,
+                           ip_address: request.remote_ip,
+                           parameters: request.filtered_parameters,
+                           timestamp: Time.current }
```

Comment:
please add the following require line at the top, so this doesn't explode
`require 'active_support/core_ext/time'`